templates/Template.NEP5.CSharp/NEP5.cs

```diff
@@ -0,0 +1,85 @@
+using Neo.SmartContract.Framework;
+using Neo.SmartContract.Framework.Services.Neo;
+using System;
+using System.ComponentModel;
+using System.Numerics;
+using System.Runtime.CompilerServices;
+
+namespace Template.NEP5.CSharp
+{
+    [Features(ContractFeatures.HasStorage | ContractFeatures.Payable)]
+    public partial class NEP5 : SmartContract
+    {
+        #region Token Settings
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static string Name() => "Token Name";
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static string Symbol() => "TokenSymbol";
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static byte Decimals() => 8;
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static ulong MaxSupply() => 1_000_000_000;
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static ulong InitialSupply() => 20_000_000;
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static string[] SupportedStandards() => new string[] { "NEP-5", "NEP-10" };
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        private static byte[] Owner() => new byte[] { 0xf6, 0x64, 0x43, 0x49, 0x8d, 0x38, 0x78, 0xd3, 0x2b, 0x99, 0x4e, 0x4e, 0x12, 0x83, 0xc6, 0x93, 0x44, 0x21, 0xda, 0xfe };
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static ulong TokensPerNEO() => 1_000_000_000;
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static ulong TokensPerGAS() => 1;
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static byte[] NeoToken() => new byte[] { 0x9b, 0xde, 0x8f, 0x20, 0x9c, 0x88, 0xdd, 0x0e, 0x7c, 0xa3, 0xbf, 0x0a, 0xf0, 0xf4, 0x76, 0xcd, 0xd8, 0x20, 0x77, 0x89 };
```

Comment:
notification.ScriptHash in contract is little-endian
```suggestion
         public static byte[] NeoToken() => new byte[] { 0x89, 0x77, 0x20, 0xd8, 0xcd, 0x76, 0xf4, 0xf0, 0x0a, 0xbf, 0xa3, 0x7c, 0x0e, 0xdd, 0x88, 0x9c, 0x20, 0x8f, 0xde, 0x9b };
```